File: 58-length-of-last-word/length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        s=s.strip()
        s=s[::-1]
        if(s.count(' ')==0):
            return len(s)
        for i in range(0,len(s)):
            if(s[i]==' '):
                return i

File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int length=s.size();
        bool word=false;
        int counter=0;
        for(int i=length-1;i>=0;i--){
            if(!word && s[i]!=' ') {
                word=true;
                counter++;
            }
            else if(word && s[i]!=' '){
                counter++;
            }
            else if(word && s[i]==' '){
                break;
            }
        }
        return counter;
    }
};